win32/src/threadpool_win32.c

```diff
@@ -265,6 +286,59 @@ void threadpool_close(THANDLE(THREADPOOL) threadpool)
     }
 }
 
+/* threadpool_create_work_item */
+
+/* Codes_SRS_THREADPOOL_WIN32_05_004: [ threadpool_create_work_item shall initialize the work_item_context to NULL, where NULL would be a failure indicator to the calling functions ] */

```

Comment:
>Codes_SRS_THREADPOOL_WIN32_05_004 [](http://example.com/codeflow?start=3&length=33)

for new code, we typically do something in the MD like change it to "S_R_S" or something so that traceability tool doesn't flag it as an error.

That is better than adding specs to the code/test which are not yet implemented.

Reply:
So we should remove all of the specs in the .c files for now until it is implemented